clang/test/Driver/suppress_LNK4221_intelfpga_sycl-link.cpp

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: system-windows
+// RUN:  %clangxx -target x86_64-windows-msvc-pc -fsycl -fintelfpga -fsycl-link -Xshardware %s -### 2>&1 \
+// RUN:  | FileCheck %s
+// RUN:  %clang_cl -target x86_64-windows-msvc-pc -fsycl -fintelfpga -fsycl-link -Xshardware %s -### 2>&1 \
```

Comment:
```suggestion
// RUN:  %clang_cl -target x86_64-pc-windows-msvc -fsycl -fintelfpga -fsycl-link -Xshardware %s -### 2>&1 \
```

Reply:
Apologies, I got my syntax messed up when I suggested the change.

Reply:
Hi, @mdtoguchi 
Fixed.
Thank you very much.